android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -893,4 +893,17 @@ private void writeAudioToFile(byte[] audio) {
             Log.e(TAG, "Failed to write audio to file e:" + e);
         }
     }
+
+    private void forceSpeakerRoute(AudioTrack audioTrack)
+    {
+        AudioManager audioManager = (AudioManager) context.getApplicationContext().getSystemService(Context.AUDIO_SERVICE);
```

Comment:
Note this does not compile, `context` is not available. `audioManager` should be used directly as it is setup during init phase. Also `AudioDeviceInfo` is not imported

Reply:
I will fix it

Reply:
I think I fixed my IDE issue now... so I can also see that